.github/workflows/ci.yml

```diff
@@ -103,16 +103,60 @@ jobs:
         with:
           path: node_modules
           key: ${{ runner.os }}-node-${{ needs.install-deps.outputs.node-version }}-modules-${{ hashFiles('package-lock.json') }}
+      - name: Run lint
+        run: npx nx affected:lint --quiet --silent --parallel=3
+
+  check-workspace:
+    name: Check workspace
```

Comment:
We could be more specific and name this step "Verify dependencies" or something.

Reply:
Done